src/registrar/templates/includes/modal.html

```diff
@@ -24,39 +25,49 @@ <h2 class="usa-modal__heading">
         <div class="usa-modal__footer">
             <ul class="usa-button-group">
                 <li class="usa-button-group__item">
-                    {% if not_form and modal_button %}
-                        {{ modal_button }}
+                    {% if cancel_button_only %}
+                        <button
+                            type="button"
+                            class="{{ modal_button_class|button_class }}"
+                            data-close-modal
+                        >
+                        {% if modal_button_text %}
+                            {{ modal_button_text }}
+                        {% else %}
+                            Cancel
+                        {% endif %}
+                    </button>
+                    {% elif modal_button_id and modal_button_text %}
```

Comment:
Add a comment that indicates that this button block triggers form submission for a form on the modal's page